pkg/cmd/node/node_ssh.go

```diff
@@ -36,6 +37,26 @@ func newCmdNodeSSH(ctx api.Context) *cobra.Command {
 				return err
 			}
 
+			if masterProxy {
+				initialClientOpts := sshclient.ClientOpts{
+					Input:  ctx.Input(),
+					Out:    ctx.Out(),
+					ErrOut: ctx.ErrOut(),
+				}
+
+				initialClientOpts.User = clientOpts.User
+				initialClientOpts.Host = clientOpts.Proxy
+				initialSSHClient, err := sshclient.NewClient(initialClientOpts, pluginutil.Logger())
+				if err != nil {
+					return err
+				}
+				err = initialSSHClient.Run([]string{"bash", "-c", "'true'"})
```

Comment:
Could you add a comment explaining that it is for backwards compatibility and that later we're passing the StrictHostKeyChecking option? As it might be hard for newcomers to understand the reasoning behind this.